codice/ddf
# [2.26.x] Dynamic rtf

#### What does this PR do?
Updates the RTF transformer to create a document based on the Column Order and Aliases arguments if provided.

#### Who is reviewing it? 
@glenhein 
@jlcsmith
@jrnorth

#### How should this be tested?
Using ddf-ui export results (1 and multiple) to RTF. Verify the attributes exported match the inspector view and any aliases.

#### Checklist:

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.
